kornia/contrib/object_detection.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+from kornia.core import Module, Tensor, concatenate, tensor
+
+
+class ObjectDetector(Module):
+    def __init__(self, model: Module, confidence_threshold: float = 0.3, postprocess_type: str = "detr"):
+        if postprocess_type.lower() != "detr":
+            raise ValueError("Currently only DETR post-process is supported")
+        super().__init__()
+        self.model = model
+        self.confidence_threshold = confidence_threshold
+        self.postprocess_type = postprocess_type
+
+    def postprocess(self, data: dict[str, Tensor], height: int, width: int) -> list[Tensor]:
+        if self.postprocess_type == "detr":
+            return self._postprocess_detr(data, height, width)
+        else:
+            raise ValueError("Currently only DETR post-process is supported")
+
+    def _postprocess_detr(self, data: dict[str, Tensor], height: int, width: int) -> list[Tensor]:
```

Comment:
should be this inside the detr architecture directory ? so that the ObjectDetection api is agnostic to that